pkg/storage/sqlstorage/aggregations.go

```diff
@@ -113,3 +114,30 @@ func (s *Store) getVolumes(ctx context.Context, exec executor, accountAddress, a
 func (s *Store) GetVolumes(ctx context.Context, accountAddress, asset string) (core.Volumes, error) {
 	return s.getVolumes(ctx, s.schema, accountAddress, asset)
 }
+
+func (s *Store) getBalancesAggregated(ctx context.Context, exec executor, q storage.BalancesQuery) (core.AssetsBalances, error) {
+	balances, err := s.GetAggregatedBalancesData(ctx, exec, q)
```

Comment:
i agree with you about the private method being enough, but i think i'll put them in the balances.go file, since i'd rather not have too much things going in a generic aggregations.go file, what do you think ?